src/components/variable.jsx

```diff
@@ -144,18 +144,18 @@ const Variable = React.createClass({
               </thead>
               <tbody>
                 {
-                Object.keys(variable.possibleValues).map((keyword) => {
-                  return (
-                    <tr>
-                      <td className="list-entry">
-                        <samp>{keyword}</samp>
-                      </td>
-                      <td>
-                        <samp>{variable.possibleValues[keyword]}</samp>
-                      </td>
-                    </tr>
-                  )
-                })
+                  Object.keys(variable.possibleValues).map((keyword) => {
+                    return (
+                      <tr key={keyword}>
```

Comment:
Is this `key` used somewhere or is it here to bring similarity between variables and parameters? 🤔

Reply:
No, in React when you loop over a list you are supposed to provide a unique key for each element. Otherwise it throws a Warning.